### PR TITLE
fix(amplify-provider-awscloudformation): fix profile loading

### PR DESCRIPTION
**Pull request by UnleashedMind**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/lib/system-config-manager.js [hunk 1]
```diff
@@ -92,10 +94,11 @@ async function getRoleCredentials(profileConfig) {
...
     ExternalId: profileConfig.external_id,
   }).promise();
-  // accessKeyId: data.Credentials.AccessKeyId,
-  // secretAccessKey: data.Credentials.SecretAccessKey,
-  // sessionToken: data.Credentials.SessionToken
-  return roleData.Credentials;
+  return {
+    accessKeyId: roleData.Credentials.AccessKeyId,
+    secretAccessKey: roleData.Credentials.SecretAccessKey,
+    sessionToken: roleData.Credentials.SessionToken,
```
Comment (kaustavghosh06) [inline packages/amplify-provider-awscloudformation/lib/system-config-manager.js]: Is it okay if the sessionToken is undefined? Did you test that case?

Comment (UnleashedMind) [reply packages/amplify-provider-awscloudformation/lib/system-config-manager.js]: no, it's not OK, I tested and it fails without the session token. 
but if the assumeRole is successful, there's always going to be data.Credentials.SessionToken.

Comment (kaustavghosh06) [reply packages/amplify-provider-awscloudformation/lib/system-config-manager.js]: Okay, so in that case, if my profile doesn't assume an STS role, will the SDK calls fail?

Comment (UnleashedMind) [reply packages/amplify-provider-awscloudformation/lib/system-config-manager.js]: If the profile does not assume role, the credentials will be plain old credential with key id and key, there's no session token. 
AWS checks if the credential is a temp credential or a normal credential, for temp credentials, it requires session token.

**Pull request merged**